docs/src/main/asciidoc/centralized-log-management.adoc

```diff
@@ -6,18 +6,15 @@ https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 = Centralized log management (Graylog, Logstash, Fluentd)
 include::_attributes.adoc[]
 :categories: observability
-:summary: This guide explains how to centralize your logs with Logstash or Fluentd using the Graylog Extended Log Format (GELF).
+:summary: This guide explains how to centralize your logs with Graylog, Logstash or Fluentd.
 :topics: observability,logging
-:extensions: io.quarkus:quarkus-logging-gelf
+:extensions: io.quarkus:quarkus-logging-gelf,io.quarkus:quarkus-opentelemetry
```

Comment:
Can you please add a warning at the top of the page that the extension is deprecated and opentelemetry-logging should be used?... with a link to it?

Reply:
I made a different editorial choice ;) 

The guide in itself is worth keeping, it talked about various ways to send logs to a centralized solution.
Before, it talked mainly about the logging-gelf extension but now it talks about:
- The socket handler with example for Elastic
- The syslog handler with example for Fluentd
- Opentelemetry Logging with a  link to the guide
- logging-gelf with a warning for deprecation and on each section a link to a replacement

So the WARNING is already there, but not at the top of the page because this is not a page about loggin-gelf but about centralized logging which in itself is not deprecated.

Reply:
makes sense